app/serializers/api/admin/headings/commodity_serializer.rb

```diff
@@ -8,15 +8,11 @@ class CommoditySerializer
 
         set_id :goods_nomenclature_sid
 
-        attributes :description, :goods_nomenclature_item_id, :goods_nomenclature_sid
+        attributes :description, :goods_nomenclature_item_id, :goods_nomenclature_sid, :producline_suffix
 
         attribute :search_references_count do |commodity|
           commodity.search_references.count
         end
-
-        attribute :leaf do |object|
```

Comment:
This is still in use in the admin area? So presumably this will result in a error on that page?

https://github.com/trade-tariff/trade-tariff-admin/blob/main/app/views/synonyms/headings/commodities/index.html.erb#L20

Should removal of this happen happen usage in the admin area has been removed?

Reply:
See https://github.com/trade-tariff/trade-tariff-admin/pull/64

Reply:
I had it bundled in another branch but will do this in isolation to enable easy release of this PR